packages/docs/src/hooks/use-showcase.ts

```diff
@@ -51,17 +75,18 @@ export const useShowcaseDemos = () => {
         })
       );
       
-      const demos = resourceRecords
+      const demos: ShowcaseDemo[] = resourceRecords
         .filter((resource): resource is AppResource => !!resource)
         .filter(({ resource }) => SHOWCASE_APPS.includes(resource.id.toString()))
         .map(({ resource, record }) => {
-          return {
-            id: resource.id.toString(),
-            // TODO(wittjosiah): Display name vs. description.
-            title: record.meta.description ?? 'Missing name',
-            description: resource.id.toString() ?? 'Missing description',
+          const id = resource.id.toString();
+          return { // TODO(zarco): Add a `imageHash` property for record inside `data`.
+            id,
+            title: record.data.displayName ?? id,
+            description: record.meta.description ?? '', // use faker sentense
```

Comment:
Why faker? We should probably just have nothing here if it's missing so it doesn't look bad, but I had it displaying as "Missing" for now so it was obvious and we could go find them all and fix them

Reply:
I guess we don't need faker right now.
That was a note from Rich on a call.